recipes.js

```diff
@@ -1,10 +1,135 @@
+// EXERCISE SETUP
+const express = require("express");
 const mongoose = require('mongoose');
-const Schema   = mongoose.Schema;
+const Schema = mongoose.Schema;
 const data = require('./data.js');
+const Recipe = require('./models/recipe-model.js');
+
 
 mongoose.connect('mongodb://localhost/recipeApp')
   .then(() => {
     console.log('Connected to Mongo!');
   }).catch(err => {
     console.error('Error connecting to mongo', err);
   });
+
+// JULES SETUP
+
+const app = express();
```

Comment:
You didn't need to add this setup because we were not running it on the server or using hbs